### PR TITLE
Add - microservices templates

### PR DESCRIPTION
**Pull request by vasilerobert85**

### CODE REVIEW

**Comment by malexsan1** [comment]
Contrats for your first contribution on GitHub!

File: generators/app/templates/microservice/gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Copy this generated service and add it to the main Docker-compose.yml file
```
Comment (malexsan1) [inline generators/app/templates/microservice/gitlab-ci.yml]: This template is for the `gitlab-ci.yml` not docker compose.

File: generators/app/templates/microservice/Dockerrun.aws.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      "Name": "496598730381.dkr.ecr.eu-west-1.amazonaws.com/service-<%= serviceName%>:latest",
+      "Update": "true"
+    },
+    "Ports": [
+      {
+        "ContainerPort": "3000"
+      }
+    ],
+    "Command": "yarn start"
+  }
```
Comment (malexsan1) [inline generators/app/templates/microservice/Dockerrun.aws.json]: Add new line at the end of file.

File: package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "generator-hindawi",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (malexsan1) [inline package.json]: Revert this file.

File: runner.yml [hunk 4]
```diff
@@ -0,0 +1,532 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (malexsan1) [inline runner.yml]: Delete this file.

**Comment by vasilerobert85** [comment]
Updated

**Pull request closed without merging**